.changeset/early-ghosts-enter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
```

Comment:
```suggestion
'@primer/react': minor
```

Small suggestion based on the versioning guidelines we have (specifically that we're adding functionality)